packages/deployed-backend-client/API.md

```diff
@@ -98,7 +98,9 @@ export enum BackendOutputClientErrorType {
     // (undocumented)
     METADATA_RETRIEVAL_ERROR = "MetadataRetrievalError",
     // (undocumented)
-    NO_OUTPUTS_FOUND = "NoOutputsFound"
+    NO_OUTPUTS_FOUND = "NoOutputsFound",
+    // (undocumented)
+    NON_EXISTENT_STACK = "ValidationError"

```

Comment:
1. Should this be `NO_STACK_FOUND` to match one above?
2. Why is this `ValidationError` where other enum members seem to just paraphrase the key ?

Reply:
1. Sounds good to keep naming consistent
2. I was basing this off of the initial error code from CloudFormation (initially the key was also `VALIDATION_ERROR`), updating the string to match the new key name